suites/reef/baremetal/eklavya_suite.yaml

```diff
@@ -0,0 +1,729 @@
+            controllers:
+              - extensa027
+            drivers:
+              count: 4
+              hosts:
+                - extensa027
+                - extensa028
+        ceph-sec:
+          config:
+            controllers:
+              - extensa034
+            drivers:
+              count: 4
+              hosts:
+                - extensa034
+                - extensa035
+
+      desc: Start COS Bench controller and driver
+      module: cosbench.py
+      name: deploy cosbench
+#configuring the secondary zone and archive zone from the Primary's Haproxy.
+
+  - test:
+      abort-on-fail: true
+      clusters:
+        ceph-pri:
+          config:
+            cephadm: true
+            commands:
+              - "ceph orch restart rgw.shared.sync.80"
+
+
+        ceph-sec:
+          config:
+            commands:
+              - "sleep 120"
+              - "radosgw-admin realm pull --rgw-realm india --url http://extensa022:5000  --access-key a123 --secret s123 --default"
+              - "radosgw-admin period pull --url http://extensa022:5000 --access-key a123 --secret s123"
+              - "radosgw-admin zone create --rgw-realm india --rgw-zonegroup shared --rgw-zone secondary --endpoints http://extensa032:80,http://extensa031:80,http://extensa033;80 --access-key a123 --secret s123"
+              - "radosgw-admin period update --rgw-realm india --commit"
+              - "ceph config set client.rgw.shared.sync.80 rgw_realm india"
+              - "ceph config set client.rgw.shared.sync.80 rgw_zonegroup shared"
+              - "ceph config set client.rgw.shared.sync.80 rgw_zone secondary"
+              - "ceph orch restart rgw.shared.sync.80"
+              - "radosgw-admin zone modify --rgw-realm india --rgw-zonegroup shared --rgw-zone secondary --endpoints http://extensa030:5000"
+              - "radosgw-admin period update --rgw-realm india --commit"
+
+      desc: Setting up RGW multisite replication environment
+      module: exec.py
+      name: setup multisite
+      polarion-id: CEPH-10362
+
+  - test:
+      clusters:
+        ceph-pri:
+          config:
+            cephadm: true
+            commands:
+              - "ceph versions"
+              - "radosgw-admin sync status"
+              - "ceph -s"
+              - "radosgw-admin realm list"
+              - "radosgw-admin zonegroup list"
+              - "radosgw-admin zone list"
+              - "radosgw-admin user list"
+        ceph-sec:
+          config:
+            cephadm: true
+            commands:
+              - "ceph versions"
+              - "radosgw-admin sync status"
+              - "ceph -s"
+              - "radosgw-admin realm list"
+              - "radosgw-admin zonegroup list"
+              - "radosgw-admin zone list"
+              - "radosgw-admin user list"
+      desc: Retrieve the configured environment details
+      polarion-id: CEPH-83575227
+      module: exec.py
+      name: get shared realm info
```

Comment:
@viduship , could you move the multisite get info test to the end of suite & comment the rest of the deployment. This would help us to run only the testcases going forward.

Reply:
@mkasturi18 sure, I have updated.

Reply:
@viduship , could you also comment the deployment of cluster.

Reply:
okay, commented the deployemnt